### PR TITLE
Fixes sample domain in image wdt

### PR DESCRIPTION
**Pull request by CarolynRountree**
Align the domain-home-in-image samples with the newest changes to the corresponding docker images. This includes executing commands relative to the directory ${scriptDir}.

### CODE REVIEW

**Comment by markxnelson** [comment]
does this rely on some change to docker-images repo too, or is it standalone?

**Comment by doxiao** [comment]
@markxnelson The PR is to align the Operator sample script with the latest changes to the docker-images repo - https://github.com/oracle/docker-images/pull/1088 has been merged already.

**Comment by CarolynRountree** [comment]
The changes to docker-images have already been merged to master by Monica.

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh [hunk 1]
```diff
@@ -246,56 +246,42 @@ function createDomainHome {
   if [ -z $imagePath ]; then
```
Comment (simon-meng-cn) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh]: It is best to move the following 11 lines of code into the createFiles function.

**Pull request merged**